src/new-components/tooltip/tooltip.module.css

```diff
@@ -1,4 +1,8 @@
+:root {
+    --reactist-tooltip-zindex: 1000;
```

Comment:
_(This is the comment for which I'm requesting changes, I do not recall the emoji we agreed to use for this)_

We already use a naming convention for zindex-related variables: `--reactist-stacking-order-*`

Also, we already have a convention of not declaring its value, and use a default value when the variable is used. See my next comment below.